drivers/virtualbox/virtualbox.go

```diff
@@ -386,6 +376,10 @@ func (d *Driver) Start() error {
 
 	switch s {
 	case state.Stopped, state.Saved:
+		d.SSHPort, err = setPortForwarding(d.MachineName, 1, "ssh", "tcp", 22, d.SSHPort)
```

Comment:
Will `1` always be the interface number?  i.e. is hardcoding this OK?  cc @SvenDowideit @ehazlett